voucher/src/test/java/com/programmers/voucher/stream/JdbcVoucherStreamTest.java

```diff
@@ -0,0 +1,107 @@
+package com.programmers.voucher.stream;
+
+import com.programmers.voucher.domain.voucher.FixedAmountVoucher;
+import com.programmers.voucher.domain.voucher.PercentDiscountVoucher;
+import com.programmers.voucher.domain.voucher.Voucher;
+import com.programmers.voucher.stream.voucher.JdbcVoucherStream;
+import com.programmers.voucher.stream.voucher.VoucherStream;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.transaction.annotation.Transactional;
+
+import javax.sql.DataSource;
+import java.util.Map;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertAll;
+
+@SpringBootTest
+@Transactional
+class JdbcVoucherStreamTest {
+
+    @Autowired
+    DataSource dataSource;
+    @Autowired
+    VoucherStream voucherStream;
+
+    @Test
+    @DisplayName("바우처를 저장하면 db에 저장되어야 함.")
+    void save() {
+        // given
+        assertThat(voucherStream.findAll()).isEmpty();
+        FixedAmountVoucher fixedAmountVoucher = new FixedAmountVoucher("skdodoll", 10000);
+        PercentDiscountVoucher percentDiscountVoucher = new PercentDiscountVoucher("testId", 80);
+
+        // when
+        voucherStream.save(fixedAmountVoucher);
+        voucherStream.save(percentDiscountVoucher);
+        // then
+        assertThat(voucherStream.findAll().size()).isEqualTo(2);
```

Comment:
한줄에 한점을 찍는다. 요기도 적용해주세요. 테스트코드에 가독성을 신경써주세요!
```suggestion
        var actual = voucherStream.findAll()
             .size();
        assertThat(actual).isEqualTo(2);
```

Reply:
앞으로 한줄에 한점 찍기 실천해 보도록 하겠습니다!